standalone/ansible/roles/common/tasks/main.yml

```diff
@@ -96,3 +97,29 @@
 - name: enable systemctl services for deploy user
   become: true
   shell: loginctl enable-linger deploy
+
+# firewalling
+- name: change default incoming policy changed to 'deny'
+  ufw:
+    direction: incoming
+    default: deny
```

Comment:
Can you shed a little more light on what specifically this disallows?

Reply:
This denies all incoming traffic (all ports). Traffic to only the explicitly `allowed` services will be allowed.

Reply:
Nice, thanks.